src/io-methods/s3.js

```diff
@@ -34,11 +45,12 @@ exports.writeManifest = function(filePath, data) {
   const jsonPromise = new Promise(function(resolve, reject) {
     const file = parseFilePath(filePath)
     s3.putObject({
-        Bucket: file.bucket,
-        Key: file.key,
-        Body: data,
-        ContentType: 'application/json',
-        CacheControl: 'public, must-revalidate, max-age=0'
+      Bucket: file.bucket,
+      Key: file.key,
+      Body: data,
+      ContentType: 'application/json',
+      CacheControl: 'public, must-revalidate, max-age=0',
+      ACL: 'public-read',
```

Comment:
@joeldenning would it make sense to make ACL configurable?

Reply:
I don’t know if a situation where public-read isn’t the right thing - would prefer to not add more configurable things unless there’s a reason to.